scripts/deploy_to_prod.sh

```
@@ -0,0 +1,28 @@
+EXECUTABLE=server  # Change if your binary has a different name
+BUILD_DIR=target/release
+PROD_DIR=prod
+
+# Ensure the executable exists
+if [ ! -f "$BUILD_DIR/$EXECUTABLE" ]; then
+  echo "Error: $BUILD_DIR/$EXECUTABLE not found. Build first with 'cargo build --release'."
+  exit 1
+fi
+
+# Clean prod directory
+rm -rf "$PROD_DIR"/*
```

Shellcheck (0.11.0)

[warning] Use "${var:?}" to ensure this never expands to /* .

(SC2115)